src/main/java/ch/njol/skript/conditions/CondIsOnline.java

```diff
@@ -33,32 +33,41 @@
  * @author Peter Güttinger
  */
 @Name("Is Online")
-@Description("Checks whether a player is online.")
+@Description("Checks whether a player is online. The 'connected' version will return false once this player leaves the server, even if they rejoin.")
 @Examples({"player is online",
-		"player-argument is offline"})
+	"player-argument is offline",
+	"while player is connected:",
+	"\twait 60 seconds",
+	"\tsend \"hello!\" to player"
+})
 @Since("1.4")
 public class CondIsOnline extends PropertyCondition<OfflinePlayer> {
 	
 	static {
-		register(CondIsOnline.class, "(online|1¦offline)", "offlineplayers");
+		register(CondIsOnline.class, "(online|:offline|:connected)", "offlineplayers");
 	}
```

Comment:
Ayham, has brought up on discord that supporting `is still connected` might be a more relatable pattern, unsure how we could include it in this, but might be a possibility.

Reply:
I'm not a fan of it but if people want it they can add it.

Reply:
Actually I think this can't/shouldn't be done because the negation pattern `if player isn't still connected` sounds rubbish